src/config/myanmar/prism.json

```diff
@@ -1,7 +1,8 @@
 {
   "country": "Myanmar",
-  "WFPAuthRequired": true,
+  "WFPAuthRequired": false,
```

Comment:
Did you mean to deactivate auhentication?

Reply:
I removed this which was probably just for testing. We'll use this feature in Cambodia first